lambda/facia-responder/src/util.test.ts

```diff
@@ -0,0 +1,21 @@
+import format from "date-fns/format";
+import {generatePublicationMessage} from "./util";
+
+describe("generatePublicationMessage", ()=>{
+  it("should generate a different message if the issueDate is today", ()=>{
+    const nowTime = new Date();
+    const formattedDateToday = format(nowTime, "yyyy-MM-dd");
+    const result = generatePublicationMessage(formattedDateToday);
+    expect(result).toEqual("This issue has been published and should be live in the app imminently");
+  });
+
+  it("should include the issue date if the issueDate is not today", ()=>{
+    const result = generatePublicationMessage("2023-02-10", new Date(2023, 1, 1));
+    expect(result).toContain("Fri, 6th Feb 2023")
```

Comment:
Not sure I understand why we'd expect Fri 6th Feb here when neither of the dates supplied are that date?

Reply:
Thanks for catching this, the format string was wrong! Fixed now.